packages/graph-retriever/src/graph_retriever/testing/adapter_tests.py

```diff
@@ -185,6 +187,8 @@ class GetAdjacentCase:
             "crab",
         ],
     ),
+    # Add test for IDs when filtered out.
```

Comment:
Some extra tests this suggested (that I think we'd currently fail in at least some stores):

1. An ID edge to an edge that doesn't match the user defined filter. I believe it shouldn't be returned.
2. Tests for a combination of ID and metadata filters.

Reply:
https://github.com/datastax/graph-rag/issues/101